geniza/footnotes/models.py

```diff
@@ -633,6 +633,23 @@ def content_html_str(self):
                 ]
             )
 
+    @cached_property
+    def content_text_canvases(self):
+        "content as a list of strings, one per canvas, used for regex search index"
```

Comment:
the regex search detail to me seems like it belongs in a comment not a docstring since we're likely to find other uses for this